server/hotrod/src/main/java/org/infinispan/server/hotrod/CacheRequestProcessor.java

```diff
@@ -661,11 +687,18 @@ void putStream(HotRodHeader header, Subject subject, byte[] key, ByteBuf buf, lo
       }
    }
 
-   void writeException(HotRodHeader header, Object span, Throwable cause) {
+   void writeException(HotRodHeader header, InfinispanSpan span, Throwable cause) {
       try {
-         telemetryService.recordException(span, cause);
+         span.recordException(cause);
       } finally {
          writeException(header, cause);
       }
    }
+
+   private InfinispanSpan requestStart(HotRodHeader header, AdvancedCache<?, ?> cache) {
+      var attributes = new InfinispanSpanAttributes.Builder(SpanCategory.CONTAINER)
+            .withCache(header.cacheName, cache.getCacheConfiguration())
```

Comment:
Looks like this line has to be changed to:
```suggestion
            .withCache(header.cacheName, SecurityActions.getCacheConfiguration(cache))
```

Reply:
applied the change thank you